src/components/Checkbox/style.scss

```diff
@@ -1,151 +1,72 @@
-    }
+  &.wrapperError .label::before,
+  &.wrapperError .label:hover::before {
+    border-color: $cWarningRed;
   }
 }
 
-.indeterminate {
-  position: absolute;
-  opacity: 0;
-  z-index: 10;
+.label {
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  flex-direction: row;
+  font-size: $typeBase;
+  justify-content: stretch;
+  line-height: 16px;
 
-  ~ .checkbox::before {
+  &::before {
+    border: 1px solid $cGray300;
+    border-radius: $inputRadius;
     content: '';
-    height: 2px;
+    margin-right: 0.3em;
+    width: 16px;
+    height: 16px;
   }
 
-  &:checked ~ .checkbox {
-    &::before {
-      display: block;
-      width: 8px;
-      height: 2px;
-      background: $cWeWorkBlack;
-      position: relative;
-      top: 0;
-    }
+  &:hover::before {
+    border-color: $cGray200;
   }
 }
 
 .checkbox {
-  width: 16px;
-  height: 16px;
-  background: $cWhite;
-  border: $checkboxBorder;
-  border-radius: 3px;
+  position: absolute;
+  clip: rect(0, 0, 0, 0);
   background: $cWhite;
-  transition: border-color 0.1s ease-out;
-  box-sizing: border-box;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  &::before {
-    display: block;
-    content: '';
-    background: none;
-  }
-}
-
-.text {
-  margin-left: 10px;
-  @extend %type--body;
-}
+  transition-duration: 0.1s;
+  transition-property: border-color, box-shadow, color;
+  transition-timing-function: ease-out;
 
-.textBold {
-  font-weight: bold;
-}
+  &:checked + .label::before {
+    background: url('data:image/svg+xml;utf8,<svg width="12px" height="10px" viewBox="0 0 12 10" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><title>checked</title><defs></defs><g id="Symbols" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g id="checkbox-selected" transform="translate(-2.000000, -8.000000)" fill="#252729"><polygon id="checked" points="6 18 2 13.8333333 3.6 12.1666667 6 14.6666667 12.4 8 14 9.66666667"></polygon></g></g></svg>') center center no-repeat;
+  }
 
-.textDisabled {
-  color: $cGray200;
-}
+  &:checked:disabled + .label::before {
+    background-image: url('data:image/svg+xml;utf8,<svg width="12px" height="10px" viewBox="0 0 12 10" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><title>checked</title><defs></defs><g id="Symbols" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g id="checkbox-selected" transform="translate(-2.000000, -8.000000)" fill="#c2c2c2"><polygon id="checked" points="6 18 2 13.8333333 3.6 12.1666667 6 14.6666667 12.4 8 14 9.66666667"></polygon></g></g></svg>');
```

Comment:
Same for this guy

Reply:
It's the same icon in a different color.